docs/conf.py

```diff
@@ -58,9 +58,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '5.0'
+version = '5.2'
```

Comment:
@jdmgoogle Updated with 5.1.2. Incidentally, noticed that the copyright value is ©2015. Didn't want to make the update in this PR, but thought y'all might want to know.